src/tiledimage.js

```diff
@@ -1023,6 +1023,18 @@ function loadTile( tiledImage, tile, time ) {
 function onTileLoad( tiledImage, tile, time, image, errorMsg ) {
     if ( !image ) {
         $.console.log( "Tile %s failed to load: %s - error: %s", tile, tile.url, errorMsg );
+        /**
+         * Triggered when a tile fails to load.
+         *
+         * @event tile-open-failed
+         * @memberof OpenSeadragon.Viewer
+         * @type {object}
+         * @property {OpenSeadragon.TiledImage} tiledImage - The tiled image of the unloaded tile.
+         * @property {OpenSeadragon.Tile} tile - The tile which has been unloaded.
+         * @property {number} time - The current time in milliseconds when the tile load began.
```

Comment:
"current time" and "when the tile load began" seem to contradict each other. Just kill the word "current".